### PR TITLE
Refactoring some code

### PR DESCRIPTION
**Pull request by naoyashiga**
Koloda is very nice! I will use Koloda when make Tinder like UI.

I refactored some code. Please review.

### CODE REVIEW

File: Pod/Classes/KolodaView/KolodaView.swift [hunk 1]
```diff
@@ -362,8 +356,7 @@ public class KolodaView: UIView, DraggableCardDelegate {
                 applyRevertAnimation(firstCardView)
             }
             
-            for index in 1..<visibleCards.count {
-                let currentCard = visibleCards[index]
+            for (index, currentCard) in enumerate(visibleCards) {
```
Comment (AEugene) [inline Pod/Classes/KolodaView/KolodaView.swift]: Notice that my enumeration starts from index 1.

File: Pod/Classes/KolodaView/KolodaView.swift [hunk 2]
```diff
@@ -230,11 +229,10 @@ public class KolodaView: UIView, DraggableCardDelegate {
                     self.moveOtherCardsWithFinishPercent(0)
                 },
                 completion: {
-                    finished in
+                    _ in
                     self.animating = false
                     
-                    for index in 1..<self.visibleCards.count {
-                        var card = self.visibleCards[index]
+                    for card in self.visibleCards {
```
Comment (AEugene) [inline Pod/Classes/KolodaView/KolodaView.swift]: Notice that my enumeration starts from index 1

File: Pod/Classes/KolodaView/KolodaView.swift [hunk 3]
```diff
@@ -418,16 +410,15 @@ public class KolodaView: UIView, DraggableCardDelegate {
...
-            if visibleCards.count > 0 {
-                
+            if !visibleCards.isEmpty {
                 loadMissingCards(missingCards)
             } else {
                 setupDeck()
                 layoutDeck()
                 
-                if ((delegate?.kolodaShouldApplyAppearAnimation(self)) != false) {
+                if let delegate = delegate?.kolodaShouldApplyAppearAnimation(self) where delegate == true {
```
Comment (AEugene) [inline Pod/Classes/KolodaView/KolodaView.swift]: Wrong naming. It would be better to call "delegate" as "shouldApply" or something like that.

**Comment by AEugene** [comment]
Thanks for helping Koloda on its way to perfection) I added some comments for your pull request.

**Comment by naoyashiga** [comment]
Thanks for review. I will modify my pull request code.

**Comment by AEugene** [comment]
Merged.

**Pull request merged**